apps/ejabberd/src/ejabberd_admin.erl

```diff
@@ -147,7 +148,17 @@ commands() ->
      #ejabberd_commands{name = reload_cluster, tags = [server],
                         desc = "Reload configuration file in the cluster",
                         module = ejabberd_config, function = reload_cluster,
-                        args = [], result = {res, restuple}}
+                        args = [], result = {res, restuple}},
+      #ejabberd_commands{name = join_cluster, tags = [server],
```

Comment:
Indentation is inconsistent with the former command (line 147/148). Tabs vs spaces issue? BTW, in the following lines - why 2 spaces, when everything around is 4 spaces?